### PR TITLE
Use ilp-node

### PR DESCRIPTION
**Pull request by michielbdejong**
* [x] add a koa server to serve statics, stats, and WebFinger + rpc end-point for peering
* [x] use https://npmjs.org/package/ilp-node version 2.1.0 for 'test my host' form
* [x] update columns of 'Hosts' tab to report host uptime for WebFinger and rpc, not ledger data
* [x] display 'Ledgers' tab as 'coming back soon' (see #15)
* [x] display 'Connectors' tab as 'coming back soon' (see #14)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# https://connector.land
+
+To run the koa server:
+
+```sh
+npm install
+node src/index.js data/stats.json data/creds.json public/ 8600
```
Comment (sharafian) [inline README.md:7]: maybe that should be a script so you can do it via `npm start`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    switch(ctx.path) {
+    case '/stats':
+      if (typeof ctx.query.test === 'string') {
+        await ilpNode.testHost(ctx.query.test)
+      }
+      ctx.type = 'json'
+      ctx.body = ilpNode.stats
+      console.log('rendered', ctx.body)
+    }
+    throw new Error('no')
```
Comment (sharafian) [inline src/index.js]: confusing error message

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    console.error(e)
+    return next()
+  }
+})
+app.use(koaStatic(publicFolder))
+app.listen(port)
+
+setInterval(() => {
+  ilpNode.testAll()
+}, 10000)
```
Comment (sharafian) [inline src/index.js]: maybe this delay should be configurable with an env variable

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+const Koa = require('koa')
+const koaStatic = require('koa-static')
+const IlpNode = require('ilp-node')
+
+if (process.argv.length < 6) {
+  throw new Error('Usage: node src/index.js data/stats.json data/creds.json public/ 8600')
+}
+
+const statsFile = process.argv[2]
```
Comment (sharafian) [inline src/index.js]: I think these should be environment variables instead of command line arguments, so it's harder to mix them up

**Review by sharafian — CHANGES_REQUESTED** [review]

**Comment by michielbdejong** [comment]
Thanks! Will apply the changes.

Preview on https://stats.connector.land/

**Review by sharafian — APPROVED** [review]

**Pull request merged**